pkgs/games/tetrio-desktop/default.nix

```diff
@@ -67,6 +69,14 @@ stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
+  postInstall =
+    let
+      tetrio-plus = callPackage ./tetrio-plus.nix { };
```

Comment:
Could you move `tetrio-plus` to the derivation arguments and have it default to this value?

Reply:
Done!